src/lib/swapper/swappers/ArbitrumBridgeSwapper/utils/helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { makeSwapErrorRight, type SwapErrorRight, TradeQuoteError } from '@shapeshiftoss/swapper'
+import type { Asset } from '@shapeshiftoss/types'
+import type { Result } from '@sniptt/monads/build'
+import { Err, Ok } from '@sniptt/monads/build'
+
+import type { ArbitrumBridgeSupportedChainId } from './types'
+import { arbitrumBridgeSupportedChainIds } from './types'
+
+export const assertValidTrade = ({
+  buyAsset,
+  sellAsset,
+}: {
+  buyAsset: Asset
+  sellAsset: Asset
+}): Result<boolean, SwapErrorRight> => {
+  if (
+    !arbitrumBridgeSupportedChainIds.includes(
+      sellAsset.chainId as ArbitrumBridgeSupportedChainId,
+    ) ||
+    !arbitrumBridgeSupportedChainIds.includes(buyAsset.chainId as ArbitrumBridgeSupportedChainId)
+  ) {
+    return Err(
+      makeSwapErrorRight({
+        message: `[ArbitrumBridge: assertValidTrade] - unsupported chainId`,
+        code: TradeQuoteError.UnsupportedChain,
+        details: { buyAsset, sellAsset },
+      }),
+    )
+  }
+
```

Comment:
might be worth adding an extra condition to throw if same chain

Reply:
2f7091b6e4